deploy.sh

```diff
@@ -0,0 +1,14 @@
+if [ "$1" == "prod" ];
+then
+bucket="prod.peeranha.io"
+npm run build:prod
+elif [ "$1" == "test" ];
+then
+bucket="test.peeranha.io"
+npm run build:test
+fi
```

Comment:
Has to fail if none of the above conditions is true. Print message that argument with environment name is required.